src/foam2dolfinx/open_foam_reader.py

```diff
@@ -103,12 +103,15 @@ def _create_dolfinx_mesh(self):
             )
         degree = 1  # Set polynomial degree
         cell = ufl.Cell(shape)
-        self.mesh_element = basix.ufl.element(
+        self.mesh_vector_element = basix.ufl.element(
             "Lagrange", cell.cellname(), degree, shape=(3,)
         )
+        self.mesh_scalar_element = basix.ufl.element(
+            "Lagrange", cell.cellname(), degree, shape=()
+        )
 
         # Create dolfinx Mesh
-        mesh_ufl = ufl.Mesh(self.mesh_element)
+        mesh_ufl = ufl.Mesh(self.mesh_vector_element)
```

Comment:
Could probably add a small inline comment here stating how the type of element for creating the mesh is not important